### PR TITLE
sample code

### PR DESCRIPTION
**Pull request by GarenJian-Intel**

### CODE REVIEW

File: examples/compress_gzip.c [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+    rc = qzInit(&session, 0);
+    if (rc != QZ_OK && rc != QZ_DUPLICATE) {
+        fprintf(stderr, "qzInit failed\n");
+        return -1;
+    }
+
+    // You can customize parameters(ie, level, pollingmode)
+    // before setup
+    params.common_params.polling_mode = QZ_BUSY_POLLING;
```
Comment (daweiq) [inline examples/compress_gzip.c:42]: Please setup the compression using 16KB chunk, L9 as Elliot requested in the email.

File: examples/compress_lz4.c [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+
+    rc = qzInit(&session, 0);
+    if (rc != QZ_OK && rc != QZ_DUPLICATE) {
+        fprintf(stderr, "qzInit failed\n");
+        return -1;
+    }
+
+    // You can customize parameters(ie, level, pollingmode)
+    // before setup
+    params.common_params.polling_mode = QZ_BUSY_POLLING;
```
Comment (daweiq) [inline examples/compress_lz4.c:42]: 16KB chunk, L9

File: examples/compress_gzip.c (hunk 1 above)
Comment (GarenJian-Intel) [reply examples/compress_gzip.c:42]: Updated

File: examples/compress_lz4.c (hunk 2 above)
Comment (GarenJian-Intel) [reply examples/compress_lz4.c:42]: Updated

**Comment by daweiq** [comment]
@GarenJian-Intel  Would you please ask Neusoft to try this patch, build and test the PR?

**Pull request closed without merging**